openedx/features/marketplace/constants.py

```diff
@@ -1,36 +1,100 @@
 from django.utils.translation import ugettext as _
 
+DJANGO_COUNTRIES_KEY_INDEX = 0
+DJANGO_COUNTRIES_VALUE_INDEX = 1
+
+MARKETPLACE_PARAM_QUERY_KEY = 'query'
+MARKETPLACE_PARAM_COUNTRY_KEY = 'country'
+MARKETPLACE_PARAM_CITY_KEY = 'city'
+
+ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY = 'delivery-services'
+ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY = 'funding'
+ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY = 'healthcare-supplies'
+ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY = 'human-resources'
+ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY = 'mentorship'
+ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY = 'online-training'
+ORGANIZATIONAL_PROBLEM_TYPE_OTHER_KEY = 'other-problem'
+ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY = 'remote-working-tools'
+
+ORGANIZATIONAL_PROBLEM_CHOICES = (
+    (ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY, 'Healthcare Supplies'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY, 'Remote Working Tools'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY, 'Online Training'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY, 'Delivery Services'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY, 'Mentorship'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY, 'Funding'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY, 'Human Resources'),
+)
+
+ORGANIZATIONAL_PROBLEM_CHOICES_VIEW = (
```

Comment:
If we remove the above tuple, please rename this to `ORGANIZATIONAL_PROBLEM_CHOICES`